changelog.md

```diff
@@ -2,7 +2,52 @@
 
 ## 2.8.0~rc1
 
+### Web applications
+* Updated strings based on translator feedback (#7057)
+* Improved redwood stream performance and testing (#7070)
+* Dependency changes:
+  * openssl rust crate from 0.10.57 to 0.10.60 (#7083)
+  * cryptography from 41.0.3 to 41.0.7 (#7086)
+  * rustix rust crate from 0.38.18 to 0.38.21 (#7114)
+
+### Operations
+* Updated copyright strings to reference 2024 (#7099)
+* Removed deprecated mitigation for CVE-2019-3462 (#7053)
+* Improved logic for installing admin tool apt dependencies in Tails (#7088)
+* Added support for Tails 6 to admin tools (#7116)
 
+### CI
+* Updated CI to verify that the demo container builds and runs (#7052)
+* Updated GCE CI machine type to c2-standard-8 (#7087)
+* Moved various CI jobs to Github Actions (#6969)
+* Fixed cargo-vet binary caching (#7065)
+* Upgraded to cargo-vet 0.9.0 (#7101)
+* Enabled dependabot for Github Actions (#7102)
+* Dependabot updates (#7105, #7104, #7108)
+* Fixed broken apt caches in staging-test-with-rebase job (#7110)
+
+### Development
+* Updated packaging logic to exclude config.py (#7014)
+* Fixed broken link in contributing.md (#7028)
+* Added option to specific git remote for backport script (#7044)
+* Updated functional tests to run under Selenium 4 (#7100)
+* Updated docker run parameters to only pass -it if a tty is available (#7098)
+* Updated rust toolchain in CI and Dockerfiles to 1.74.1 (#7091)
+* Decreased cargo audit error threshold (#7083)
+* Fixed hot reload functionality in dev environment (#7120)
+* Dependency changes:
+  * MarkupSafe from 2.0.2 to 2.1.2 (#7006)
```

Comment:
MarkupSafe and jinja2 are prod dependencies that should probably be in the above section?

Reply:
Yup, will add them for the next revision.